src/pages/fr/core-concepts/project-structure.md

```diff
@@ -0,0 +1,105 @@
+
+### `src/components`
+
+**Les composants** sont généralement du code réutilisables pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants frontend](/fr/core-concepts/framework-components/) comme React ou Vue. Il est commun de grouper et d'organiser tous les composants de votre projet dans ce dossier.
+
+C'est une convention commune dans les projets Astro, mais c'est pas obligatoire. Organisez vos composants comme vous le voulez, si vous le souhaitez !
+
+### `src/layouts`
+
+[Les composants Layouts](/fr/core-concepts/layouts/) sont des types de composants particuliers qui s'appliquent à des pages. Ils sont généralement utilisés par les [Pages Astro](/fr/core-concepts/astro-pages/) et les [Pages Markdown](/fr/guides/markdown-content/) pour définir leur mise en page.
+
+Comme `src/components`, organisez vos composants Layouts comme vous le souhaitez, ils ne sont pas obligatoires.
+
+### `src/pages`
+
+[Les composants Pages](/fr/core-concepts/astro-pages/) sont des types de composants particuliers utilisés pour créer de nouvelles pages. Une Page peut être un composant Astro (`.astro`) ou un fichier Markdown (`.md`) qui représente une page de contenu pour votre site.
+
+> ⚠️ `src/pages` les un dossier **obligatoire** dans votre projet Astro. Sans ça, votre site n'aura aucune pages ou routes !
+
+### `src/styles`
+
+C'est une convention commune de stocker vos fichiers CSS ou Sass dans le dossier `src/styles`, mais c'est pas obligatoire. Du moment que vos fichiers de style sont quelque pars dans le dossier `src/`, Astro va les gérer et les optimiser.
+
+### `public/`
+
+Le dossier `public/` est là pour les fichiers et les ressources qui ne doivent pas être traités durant le processus de compilation d'Astro. Ces fichiers seront copiés dans le dossier de compilation sans modification.
+
+Cet interaction fait en sorte que `public/` soit idéal pour les ressources communes comme des images, des polices d'écriture, ou des fichiers spéciaux comme `robots.txt` et `manifest.webmanifest`.
+
+Vous pouvez placer des fichiers CSS et JavaScript dans le dossier `public/`, mais gardez à l'esprit que ces fichiers ne seront pas regroupés ou optimisés dans votre build final.
+
+> 💡 *En règle générale, tout CSS ou JavaScript que vous ajoutez devrait être mis dans le dossier `src/`*
+
+### `package.json`
+
+C'est un fichier utilisé par les gestionnaires de paquets JavaScript pour gérer vos dépendances. Il définit également les scripts qui sont utilisés pour exécuter Astro (ex: `npm start`, `npm run build`).
+
+Pour plus d'aides pour créer un nouveau fichier `package.json` pour votre projet, consultez les instructions de [configuration manuelle](/fr/install/manual/).
```

Comment:
"Pour plus d'aides pour créer..." sounds very anglicisme to me. (Please don't trust me too much; I'm not a native speaker.) I would expect something more like "Pour plus d'information/d'indications en ce qui concerne la création d'un nouveau fichier...". Maybe I'm overthinking it. :)